web-components/src/components/combobox/ComboBox.ts

```diff
@@ -59,6 +59,7 @@ export namespace ComboBox {
     @property({ type: Boolean, reflect: true, attribute: "search-trim-space" }) trimSpace = false;
     @property({ type: Boolean, reflect: true }) invalid = false;
     @property({ type: String, reflect: true, attribute: "invalid-text-i18n" }) invalidText = "";
+    @property({ type: String }) ariaLabel = "";
```

Comment:
When there are properties with camelCase, we want to add an attribute value with it in dashes like so
`@property({ type: String, attribute: "aria-label" }) ariaLabel = "";`